jsr352/core/src/main/java/org/hibernate/search/jsr352/massindexing/impl/util/CompositeIdOrder.java

```diff
@@ -113,10 +119,21 @@ private Criterion restrictLexicographically(BiFunction<String, Object, SimpleExp
 			// The last expression has whatever symbol is defined by "lastRestriction"
```

Comment:
My trivial comment: should we update the `"lastRestriction"` here? The variable is renamed to `"strictOperator"` now